### PR TITLE
Add new constants directory to hold rule configurations

### PR DESCRIPTION
**Pull request by npinciak**
I think we could organize the rules a little bit better by introducing a directory that holds all of the static/constant rule configurations.

This is request does not modify any existing imports or logic that affects the package. This just adds new files to be used in followup pull requests

### CODE REVIEW

**Comment by BenBrostoff** [comment]
Looks great! Thank you for the PR.

**Comment by BenBrostoff** [comment]
If you don't mind just fixing the unused imports from https://github.com/BenBrostoff/draftfast/actions/runs/7263813162/job/19803128017?pr=211 that would be awesome. If they're just imported to be exposed I'm fine doing the `assert <module>` solution

File: draftfast/constants/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+from roster_size import ROSTER_SIZE_BY_SITE_BY_SPORT
+from positions import POSITIONS_BY_SITE_BY_LEAGUE
```
Comment (BenBrostoff) [inline draftfast/constants/__init__.py:2]: I think these also need `assert`s to pass the build

Comment (npinciak) [reply draftfast/constants/__init__.py:2]: i guess i should have just run the linting locally!

**Comment by BenBrostoff** [comment]
Thanks again! Future PRs good anytime :-)

**Pull request merged**